cmds/report.c

```diff
@@ -260,6 +260,7 @@ static void report_functions(struct uftrace_data *handle, struct uftrace_opts *o
 
 	print_line(&output_fields, field_space);
 	print_and_delete(&sort_root, true, NULL, print_function, field_space);
+	clear_report_calls();
```

Comment:
Don't we need it for tui as well?

Reply:
I didn't apply new fields to tui yet. I think it will be my next work.

Reply:
Sounds good.